CHANGELOG.md

```diff
@@ -20,6 +20,34 @@ This document is formatted according to the principles of [Keep A CHANGELOG](htt
 
 ### Fixed
 
+## [v0.11.0-rc1]
+
+### Added
+- Created a simple example for a custom formatter ([330](https://github.com/cucumber/godog/pull/330) - [lonnblad])
```

Comment:
Why is name of the Authors necessary? To know who to blame? ;)

Reply:
haha, it has been like this since before time ;)
nah, but I believe the idea is to highlight the great work from different contributors!

Reply:
Thats nice. Hope it is not used the other way around :)